packages/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -0,0 +1,287 @@
+<template>
+  <KPop
+    class="analytics-timepicker"
```

Comment:
See my comment in the style block
```suggestion
    class="k-datetime-picker"
```